src/index.html

```diff
@@ -5,8 +5,313 @@
+          class="icon__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__li">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link"  href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__li">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link"  href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link"  href="#photo">Photo</a>
+          </li>
+          <li class="nav__li">
+            <a class="nav__link"  href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-text">Reviews: 5 </p>
+        </div>
+        <p class="card__price-text">
+          Price: <strong class="card__price">$2,199</strong>
+        </p>
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-text">Reviews: 5 </p>
+        </div>
+        <p class="card__price-text">
+          Price: <strong class="card__price">$2,199</strong>
```

Comment:
Looks like it should be a modifier
```suggestion
          Price: <strong class="card__price-text--strong">$2,199</strong>
```